charts/elasticsearch-curator/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Do you reckon is the case of adding a short `README.md` with the command to install the chart and maybe other details as we do in others charts (e.g. [rstudio helm chart's README.md](https://github.com/ministryofjustice/analytics-platform-ops/blob/master/charts/rstudio/README.md))?